### PR TITLE
Implement `use-generic-terminal-name`

### LINKED ISSUE #146: Custom terminal aliases
Hello team!

It would be great if the extension could set a custom gsettings option to change the alias of the terminal that appears in the context menu.

For example, as a distribution maintainer I want to ship Ptyxis terminal, but would prefer it to be called just "Terminal" in the context menus so it's more understandable for the user.
This could be an option like `terminal-alias` or `terminal-custom-name`.  If not set, use "Open in Ptyxis".  If it is set, the menu should display "Open in \<ALIAS>".

image

### PR DESCRIPTION
**Pull request by lvxnull**
Partially fixes #146

### CODE REVIEW

File: nautilus_open_any_terminal/nautilus_open_any_terminal.py [hunk 1]
```diff
@@ -226,27 +227,28 @@ def menu_item_id(*, foreground: bool, remote: bool):
     return f"OpenTerminal::open{'_' if foreground else '_bg_'}{'remote' if remote else 'file'}_item"
 
 
-def get_menu_items(file: FileManager.FileInfo, callback, foreground: bool):
+def get_menu_items(file: FileManager.FileInfo, callback, *, foreground: bool, terminal_name: str | None = None):
     items = []
     remote = file.get_uri_scheme() in REMOTE_URI_SCHEME
+    terminal_name = terminal_name or terminal_data.name
```
Comment (Stunkymonkey) [inline nautilus_open_any_terminal/nautilus_open_any_terminal.py:233]: can we use a different variable name for this?
I know that python had some linter issues with reassigning to the same naming but different types.

```suggestion
    terminal_str = terminal_name or terminal_data.name
```

do not forget the var below

**Review by Stunkymonkey — COMMENTED** [review]
LGTM

File: nautilus_open_any_terminal/nautilus_open_any_terminal.py (hunk 1 above)
Comment (lvxnull) [reply nautilus_open_any_terminal/nautilus_open_any_terminal.py:233]: would declaring the type explicitly also work?

Comment (Stunkymonkey) [reply nautilus_open_any_terminal/nautilus_open_any_terminal.py:233]: yeah might be sufficient as well.

Comment (lvxnull) [reply nautilus_open_any_terminal/nautilus_open_any_terminal.py:233]: this shouldn't be a problem since string can be assigned to `str | None` just fine

Comment (lvxnull) [reply nautilus_open_any_terminal/nautilus_open_any_terminal.py:233]: making a new variable here looks really ugly, i'd rather just keep this as it is and if a linter can't handle assigning a default value to a function parameter then it's broken, i shouldn't have to work around someone else's broken software

**Review by Stunkymonkey — APPROVED** [review]

**Pull request merged**